pkg/actuators/machine/actuator.go

```diff
@@ -421,7 +421,7 @@ func (a *Actuator) Update(context context.Context, cluster *clusterv1.Cluster, m
 		return a.handleMachineError(machine, err, updateEventAction)
 	}
 	// Get all instances not terminated.
-	existingInstances, err := getExistingInstances(machine, client)
+	existingInstances, err := a.getMachineInstances(cluster, machine)
 	if err != nil {
 		glog.Errorf("%s: error getting existing instances: %v", machine.Name, err)
```

Comment:
Is this log and `existingInstances` and other stuff in this function stale now?  It's not clear to me what this commit was attempting to accomplish...

Reply:
Ah, motivation [here][1], and `getMachineInstances` is [restricted to existing instances][2].  Sorry for the noise.

[1]: https://github.com/openshift/cluster-api-provider-aws/pull/271#discussion_r347829505
[2]: https://github.com/openshift/cluster-api-provider-aws/blob/65c621aa4393cc4a70f0ee90a9f2473a70276047/pkg/actuators/machine/actuator.go#L547-L557